npm/ipsm/ipsm.go

```diff
@@ -633,12 +647,16 @@ func (ipsMgr *IpsetManager) DestroyNpmIpsets() error {
 		entry.set = ipsetName
 		if _, err := ipsMgr.run(entry); err != nil {
 			metrics.SendErrorLogAndMetric(util.IpsmID, "{DestroyNpmIpsets} Error: failed to destroy ipset %s", ipsetName)
+		} else {
+			metrics.NumIPSets.Dec()
 		}
 	}
 
 	return nil
 }
 
+/* (TODO) commenting function as it not being used today.
```

Comment:
don't keep dead code

Reply:
I was following the precedent [here](https://github.com/Azure/azure-container-networking/blob/83b62409335cb6196a2b2d8729469c613fbfbcb0/npm/podController.go#L57). What do you think?

Reply:
Ok to delete `getPodObjFromNpmPodObj` as well.